backend/src/test/java/org/eclipse/tractusx/semantics/hub/ModelsApiTest.java

```diff
@@ -923,4 +923,15 @@ public void testDependentModelBAMM() throws Exception {
    private static String toMovementUrn(String urn){
       return urn + "Movement";
    }
+
+   @Test
+   public void testGetModelByURNWithInvalidURN() throws Exception {
+      String urnPrefix = "urn:invalid";
+      mvc.perform( MockMvcRequestBuilders.get( "/api/v1/models/{urn}", urnPrefix ).with( jwtTokenFactory.allRoles() ) )
+            .andDo( MockMvcResultHandlers.print() )
+            .andExpect( status().is4xxClientError() )
```

Comment:
Which error code is thrown: 4XX or 500 ? 
The description is: 500 on /models.

Reply:
Hi @tunacicek , 

We are throwing bad request - HTTP error code 400 now.
This is the expected error code in the bug - https://github.com/eclipse-tractusx/sldt-semantic-hub/issues/116